aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -10,8 +10,39 @@
 #include <cstdlib>
 #include <cstring>
 
+#ifdef HAVE_ZVECTOR_CPU_DEFINITION
+#include <fstream>
+#include <regex>
+#include <string>
+#endif
+
 namespace at::native {
 
+#ifdef HAVE_ZVECTOR_CPU_DEFINITION
+static bool proc_cpuinfo_field_contains(const std::string &field_name, const std::string &value)
+{
+  std::ifstream cpuinfofile("/proc/cpuinfo");
```

Comment:
How portable is this?  What about Windows?

Reply:
Windows seems not have any version for IBM CPU.